src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ClientGameOverSystem.java

```diff
@@ -92,6 +92,8 @@ public void onGameOver(GameOverEvent event, EntityRef entity) {
     private void addPlayerStatisticsInfo(DeathScreen deathScreen, GameOverEvent event) {
         MigLayout spadesTeamMigLayout = deathScreen.find("spadesTeamPlayerStatistics", MigLayout.class);
         MigLayout heartsTeamMigLayout = deathScreen.find("heartsTeamPlayerStatistics", MigLayout.class);
+	spadesTeamMigLayout.removeAllWidgets();
```

Comment:
Uh oh watch out! A tab character! 😱 

Protip: Make sure your IDE has the Checkstyle plugin installed and with the Terasology rules active. See https://github.com/MovingBlocks/Terasology/wiki/Checkstyle for details

Then just habitually check files you've modified in the IDE and it'll catch issues like this :-)

Reply:
yeah I was going to add this but forgot about it :grin:

Reply:
![Screenshot (394)](https://user-images.githubusercontent.com/52852877/92197831-7abb0680-ee90-11ea-9afe-fc9c3717e42b.png)
I wonder where that tab character sneaked in from, it looks fine when I view the file

Reply:
Checkstyle verified that there was no style errors in this file at least :relieved: